packages/attention/src/component.tsx

```diff
@@ -92,7 +93,7 @@ export function Attention(props: AttentionProps) {
         attentionEl.current = v;
       },
       get targetEl() {
-        return targetEl?.current;
+        return targetElRef.current;
```

Comment:
targetElRef can be null so maybe do `targetElRef?.current` just so the null is handled

Reply:
Ah good catch! 👍